assets/src/edit-story/app/rightClickMenu/constants.js

```diff
@@ -48,6 +54,46 @@ export const RIGHT_CLICK_MENU_SHORTCUT_LABELS = {
     'web-stories'
   ),
   DELETE: _x('Delete', 'The keyboard key "Delete"', 'web-stories'),
+  COMMAND_OPEN_BRACKET: _x(
+    'Command Open Bracket',
+    'The keyboard keys "Command" and "Open Bracket"',
```

Comment:
Does this need to be ctrl on linux / windows?

Reply:
There's two different ones 😄 [`COMMAND_OPEN_BRACKET` and `CONTROL_OPEN_BRACKET`](https://github.com/google/web-stories-wp/pull/8382/files/f93e2a715756e4e680c916fabe91a8d305669d05#diff-d448253e3cf32f60ad1f345c3bbf87ff388746af5a81e6c1d2b77b758239948fR57-R66)

We did this so that the translators weren't just swapping out one word